### PR TITLE
fix(xmlReader): empty files & tags doesnt crash anymore

### PR DESCRIPTION
**Pull request by Skraye**

### CODE REVIEW

**Comment by Skraye** [comment]
fixes #64

File: src/main/java/io/kestra/plugin/serdes/xml/XmlReader.java [hunk 1]
```diff
@@ -32,6 +33,7 @@
 @Schema(
     title = "Read a XML file and write it to an ion serialized data file."
 )
+@Slf4j
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/serdes/xml/XmlReader.java:36]: Can you fix the query property doc (even though out of scope it doesn't take much)

File: src/test/java/io/kestra/plugin/serdes/xml/XmlReaderWriterTest.java [hunk 2]
```diff
@@ -147,4 +147,18 @@ void ion() throws Exception {
...
+    void readEmpty() throws Exception {
+        File sourceFile = SerdesUtils.resourceToFile("xml/empty.xml");
+        this.reader(sourceFile, "/random/stuff");
+    }
+
+    @Test
+    // Assert that there is no exception throw when reading an empty file
+    void readEmptyTag() throws Exception {
+        File sourceFile = SerdesUtils.resourceToFile("xml/empty-tag.xml");
+        this.reader(sourceFile, "/random/stuff");
```
Comment (brian-mulier-p) [inline src/test/java/io/kestra/plugin/serdes/xml/XmlReaderWriterTest.java:162]:
```suggestion
        this.reader(sourceFile, "/catalog");
        String tagContent = new BufferedReader(new 
 InputStreamReader(runContextFactory.of().uriToInputStream(reader.getUri()))).lines().collect(Collectors.joining("\n"));
        assertThat(tagContent, is("\"\"");
```

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]
little assertion swap and doc enhancement and LGTM

**Pull request merged**